web/public/js/controllers/user-profile-controller.js

```diff
@@ -27,6 +27,7 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
   $scope.editMode = false;
   $scope.publicMode = false;
   $scope.publicChampion = false;
+  $scope.loggedInUserIsMemberOfDojoChampion = false;
```

Comment:
To be exact, is it checking that the user's profile being viewed is a member of a dojo that the currently logged in user is a champion of?